test/unit/api/python/test_solver.py

```diff
@@ -388,6 +415,34 @@ def test_mk_bit_vector(solver):
     assert solver.mkBitVector(8, "-1", 10) ==\
             solver.mkBitVector(8, "FF", 16)
 
+def test_mk_finite_field_elem(solver):
```

Comment:
This is missing a check for solver.mkFiniteFieldElem with an int value and a base other than 10 (which should raise a RuntimeError.